GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -104,7 +146,45 @@ - (void)verifyAccountDetailsInteractivelyWithOptions:(GIDSignInInternalOptions *
                 format:@"Your app is missing support for the following URL schemes: %@",
      [unsupportedSchemes componentsJoinedByString:@", "]];
   }
-  // TODO(#397): Start the incremental authorization flow.
+  NSURL *redirectURL = [NSURL URLWithString:[NSString stringWithFormat:@"%@:%@",
+                                             [schemes clientIdentifierScheme],
+                                             kBrowserCallbackPath]];
+
+  NSMutableDictionary<NSString *, NSString *> *additionalParameters = [@{} mutableCopy];
+  additionalParameters[kIncludeGrantedScopesParameter] = @"true";
```

Comment:
IIRC, I don't think we want to set this as `true` because that will mean we'll get back a token with _all_ scopes set to expire in the shorter timeframe set for VwG tokens. Probably good for you to double check. Thanks!

Reply:
Double checked, so I went ahead and removed setting this parameter since it's `false` by default.